docs/asset-store/assets/example-openapi-service.yaml

```diff
@@ -0,0 +1,128 @@
+  /validate:
+    post:
+      description: Represents the `validation webhook`.
+      tags:
+        - validate
+      summary: Validates input files.
+      operationId: validateReq
+      requestBody:
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              properties:
+                parameters:
+                  description: Additional parameters for the service
+                  type: string
+                content:
+                  description: File to validate
+                  type: string
+                  format: binary
+      responses:
+        "200":
+          $ref: "#/components/responses/successValidationResponse"
+        "422":
+          $ref: "#/components/responses/unprocessableEntity"
+  /extract:
+    post:
+      description: Represents the `metadata webhook`.
+      tags:
+        - extract
+      summary: Extracts metadata from input files.
+      operationId: extractReq
+      requestBody:
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              additionalProperties:
+                type: string
+                format: binary
+      responses:
+        "200":
+          $ref: "#/components/responses/successResponse"
+servers:
+  - url: http://localhost:3000/v1
+components:
+  responses:
+    successConversionResponse:
+      description: Successful response with the new file content
+      content:
+        text/plain:
+          schema:
+            type: string
+            example: "{\"info\":{\"version\":\"1.0.0\",\"title\":\"AnyOf example\"},\"topics\":{\"test\":{\"publish\":{\"$ref\":\"#\/components\/messages\/testMessages\"}}},\"asyncapi\":\"1.1.0\",\"components\":{\"messages\":{\"testMessages\":{\"payload\":{\"anyOf\":[{\"$ref\":\"#\/components\/schemas\/objectWithKey\"},{\"$ref\":\"#\/components\/schemas\/objectWithKey2\"}]}}},\"schemas\":{\"objectWithKey2\":{\"type\":\"object\",\"properties\":{\"key2\":{\"type\":\"string\"}}},\"objectWithKey\":{\"type\":\"object\",\"properties\":{\"key\":{\"type\":\"string\"}}}}}}"
+    notModified:
+      description: File content was not modified
+    successValidationResponse:
+      description: Successful response
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              data:
+                description: Files that were successfully processed
+                type: array
+                items:
+                  $ref: "#/components/schemas/ResultSuccess"
+    unprocessableEntity:
+      description: Unprocessable entities. File validation failed.
+      content:
+        text/plain:
+          schema:
+            type: string
+            example: Invalid property
+    successResponse:
+      description: Successful response
+  schemas:
+    ResultSuccess:
+      description: "It represents successful processing of a single file."
```

Comment:
```suggestion
      description: It represents successful processing of a single file.
```

Reply:
I changed it into `description: Successful processing of a single file`